doc/conf.py

```diff
@@ -64,7 +64,8 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = ['sphinx.ext.intersphinx',
-              'pyi_sphinx_roles']
+              'pyi_sphinx_roles',
+              'sphinx.ext.autodoc',]
```

Comment:
```suggestion
              'sphinx.ext.autodoc']
```
Appease the linter.

Reply:
Well I think I'm going to revert this because the rest of the docs doesn't use `autodoc`. Although I wish it did...

Reply:
@bwoodsend no, don't revert this. I'd like to transition to autodoc in the long run.